Signal/src/ViewControllers/DebugUITableViewController.m

```diff
@@ -67,12 +67,29 @@ + (void)presentDebugUIForThread:(TSThread *)thread
                                                       }],
                                   ]]];
 
-    [contents addSection:[OWSTableSection sectionWithTitle:@"Print to Debug Log"
-                                                     items:@[ [OWSTableItem itemWithTitle:@"Print all sessions"
-                                                                              actionBlock:^{
-                                                                                  [[TSStorageManager sharedManager]
-                                                                                      printAllSessions];
-                                                                              }] ]]];
+    [contents
+        addSection:[OWSTableSection
+                       sectionWithTitle:@"Session State"
+                                  items:@[
+                                      [OWSTableItem itemWithTitle:@"Print all sessions"
+                                                      actionBlock:^{
+                                                          dispatch_async([OWSDispatch sessionStoreQueue], ^{
+                                                              [[TSStorageManager sharedManager] printAllSessions];
```

Comment:
I wonder if we should make the usage of `sessionStoreQueue` an implementation detail of `TSStorageManager` and modify its methods to take completion handlers.